server/Makefile

```diff
@@ -1,6 +1,6 @@
-SERVER_DIR:=./
+SERVER_DIR := $(shell pwd)
 BIN_DIR := $(SERVER_DIR)/bin
-SERVER_NAME := server
+SERVER_NAME := bastionai_app
```

Comment:
Will these changes affect the way we build the server currently?

Reply:
Not much.
We just have to rename the package in the Cargo.toml file

Reply:
Sounds good